src/components/RedeemPopup.tsx

```diff
@@ -0,0 +1,22 @@
+import { BottomPaper } from "@creativesolutionsgroup/bottom-paper"
+import { Button, Paper, Typography } from "@mui/material"
+import { PropsWithChildren, useEffect } from "react"
+import { IReward } from "../models/User"
+
+interface PopupProps { reward: IReward, show?: boolean, onClose: Function };
+
+export const RedeemPopup = ({ reward, show, onClose }: PropsWithChildren<PopupProps>) => {
+  useEffect(() => {
```

Comment:
Can you explain lines 10 & 11 of this useEffect() to me please? I am confused on the show ? [text] : [text] portion.
I understand the show ?: boolean on line 6 indicates the property is optional (ie. it may be defined or undefined) and in lines 15-18 the text on the left hand side of ?? is to be displayed unless undefined in which the right hand side of ?? is to be displayed (similar to a default status).